### PR TITLE
Milestone 4: Features 4th pull. Responsive layout.

### PR DESCRIPTION
**Pull request by carloshs1994**
##**In this milestone.**

- Add the 1100px breakpoint for big screens (designed for 1440px)

1. 1200px breakpoint to avoid overlap
2. 950px and 768px breakpoints to use the new space

- Fix issues with static layout (use of `px` instead of `%`)

### CODE REVIEW

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @carloshs1994, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost-there-mario


## Required Changes ♻️
- [x] Make sure that the students Desktop version looks like the one in the design they chose.
- Figma template 👇  
students version
- Your version 👇 
original

- [x] Your desktop project has a README for the mobile version project. Kindly make changes to the title, also it would be better if you provide the issues page and license links.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -167,23 +171,24 @@ <h2 class="about-title">About me</h2>
...
     <hr class="footer-social-network">
 
     <ul class="footer-Links">
         <li><a href="#"><img src="assets/github-dark.png" alt="github" ></a></li>
-        <li><a href="#"><img src="assets/linkedin-dark.png"
-                    alt="linkedin"></a></li>
+        <li><a href="#"><img src="assets/linkedin-dark.png" alt="linkedin"></a></li>
         <li><a href="#"><img src="assets/hand-dark.png" alt="idk"></a></li>
         <li><a href="#"><img src="assets/twitter-dark.png" alt="twitter"></a></li>
         <li><a href="#"><img src="assets/m-dark.png" alt="M"></a></li>
```
Comment (MrBrN197) [inline index.html:194]: - [ ] [OPTIONAL] You've designed this entire page very well 👏. I want you to notice that the same separator at the bottom (`.footer-links`). that separator is supposed to span the entire page on desktop (or on all screen sizes). The previous reviewer didn't mention it so I've made it optional for you. please go ahead and make this change.
## Your Design 
image

## Figma Design
image

**Review by MrBrN197 — APPROVED** [review]
## Hi, @carloshs1994  👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (carloshs1994) [reply index.html:194]: Thanks! I'll do it 👍🏻

**Pull request merged**